README.md

```diff
@@ -807,8 +808,25 @@ $ cabal update
 $ cabal build
 ```
 
+##### Flakes support
+
+If you are using nix 2.4 style command (enabled by `experimental-features = nix-command`),
+you can use `nix develop` instead of `nix-shell` to enter the development shell. To enter the shell with specific GHC versions:
```

Comment:
IMO it'd be better if there is some explanation why we don't have GHC 8.6.5 here. Maybe it's obvious for nix users though...

Reply:
GHC 8.6.5 packages set is no longer supported in the `nixpkgs-unstable`, but we are able to obtain it from `nixos-20.09`.

Reply:
Yeah, I mean it's better to be written somewhere for nix newcomers